ecl/hqlcpp/hqlcppds.cpp

```diff
@@ -4373,6 +4373,14 @@ void HqlCppTranslator::buildRowAssign(BuildCtx & ctx, BoundRow * targetRow, IHql
             buildStmt(ctx, expr->queryChild(0));
             buildRowAssign(ctx, targetRow, expr->queryChild(1));
             return;
+        case no_selectnth:
```

Comment:
Still generally valid.  I opened https://track.hpccsystems.com/browse/HPCC-13252 to cover improving it.